.bazelrc

```diff
@@ -27,6 +27,10 @@ build --cxxopt=-std=c++14
 # https://bazel.build/docs/user-manual#strip
 build --strip=never
 build --per_file_copt=^lte/gateway/c/.*$@-g
+# Exclude a couple of warnings from compiling external libraries (note: the external folder
+# does not belong to the magma repo and is only existing for a build and contains the external
```

Comment:
Nit: Instead of `and is only existing for`: `and only exists for`

Reply:
pushed